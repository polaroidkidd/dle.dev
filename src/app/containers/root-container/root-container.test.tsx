import { render } from '@testing-library/react';
import pretty from 'pretty';
import React from 'react';
import { unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';
import RootContainer from './root-container';

let container;
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting

  if (container) {
    unmountComponentAtNode(container);
    container.remove();
  }
  container = null;
});

it('should render the root container', () => {
  act(() => {
    render(<RootContainer/>, container);
  });

  expect(pretty(container.innerHTML)).toMatchInlineSnapshot(
      `""`,
  ); /* ... gets filled automatically by jest ... */
});
