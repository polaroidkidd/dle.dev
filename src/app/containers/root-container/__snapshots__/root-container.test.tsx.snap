// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render tertiary heading with text 1`] = `
.c3 {
  font-size: 5rem;
  font-family: Poppins,sans-serif;
  font-weight: 100;
  color: #0BD29E;
}

.c2 {
  -webkit-clip-path: circle(12rem at 50% 50%);
  clip-path: circle(12rem at 50% 50%);
  width: 25rem;
  margin-top: 2rem;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-clip-path: circle(12.2rem at 50% 50%);
  clip-path: circle(12.2rem at 50% 50%);
  width: 26rem;
  background-color: $color-primary-border;
  padding-bottom: 2rem;
}

.c0 {
  margin: 2rem 1rem;
  display: grid;
  justify-items: center;
  grid-template-columns: 1fr;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <img
      alt="Profile Picture"
      className="c2"
      src="daniel_einars-400x500.jpg"
    />
  </div>
  <div
    className="c3"
  >
    dle
  </div>
</div>
`;
